.azure-pipelines.yml

```
@@ -79,6 +79,7 @@ jobs:
   steps:
   - template: tools/ci/azure/checkout.yml
   - script: |
+      set -e; set -o pipefail
```

```suggestion
      set -euo pipefail
```

Let's do `set -u` by the way. And they can be in a single line.